TimeFormat.java

```diff
@@ -1,4 +1,4 @@
-// Represents the hh:mm time format using an AM/PM format. 
+
 public class TimeFormat {

```

Comment:
The program handles hours == 12 correctly but doesn't account for 00:00, which should be formatted as 12:00 AM
Like this:
if (hours == 0) {
    hours = 12;
}